tests/rptest/services/redpanda.py

```diff
@@ -259,7 +260,7 @@ def stop_node(self, node):
 
     def clean_node(self, node):
         node.account.kill_process("redpanda", clean_shutdown=False)
-        node.account.remove(f"{RedpandaService.PERSISTENT_ROOT}/*")
+        node.account.remove(f"{RedpandaService.PERSISTENT_ROOT}/data/*")
```

Comment:
what's happening here? if removing the `data` subdir is done because something in its parent directory should remain then this seems to longer be correct for `clean_node` method and perhaps something else needs to be done for a special case?